src/components/Menubar.js

```diff
@@ -11,7 +11,7 @@ class Menubar extends Component {
     constructor(props) {
         super(props)
         this.state = {
-            activeItem: 'dashboard'
+            activeItem: window.location.pathname
```

Comment:
I think a more cleaner code could be `activeItem: window.location.pathname.substring(1) || 'dashboard'`

Reply:
This might not work as window.location.pathname returns string without slash during internal navigation. 
What about `window.location.pathname[0] == '/' ? window.location.pathname.substr(1) : window.location.pathname;`

Reply:
@jeeva2812 Can you please elaborate why will it not work as I applied those changes locally and it s working as expected.

Reply:
My bad. It works. I have made the required changes. It should work fine after making the necessary changes in the backend.